### ISSUE TITLE
Standard SQL table pattern doesn't find all table references

### ISSUE THREAD

**Issue by jared-martin** [servian/bigquery-view-analyzer#4]
The the table pattern [here](https://github.com/servian/bigquery-view-analyzer/blob/1b8532a4447e56f92b5d368cfe4a253d0ca89828/src/bigquery_view_analyzer/analyzer.py#L11) doesn't catch references like this, which are also valid in standard SQL:

```
`some-project-id`.some_dataset_id.some_table_id
```

This pattern works (note the project ID has to be there):

```python
>>> re.findall(
...     r'`(?P<project>.+?)`\.(?P<dataset>\w+)\.(?P<table>\w+)',
...     '`some-project-id`.some_dataset_id.some_table_id')
[('some-project-id', 'some_dataset_id', 'some_table_id')]
```

**Comment by christippett** [comment]
Thanks @jared-martin, I'll look to release an updated version with this change ASAP.
